docs/examples/kubo-as-a-library/go.mod

```diff
@@ -7,7 +7,7 @@ go 1.18
 replace github.com/ipfs/kubo => ./../../..
 
 require (
-	github.com/ipfs/go-libipfs v0.1.0
+	github.com/ipfs/go-libipfs v0.1.1-0.20230117180222-442a2693ba27
```

Comment:
Pointing at commit from master because we can't do release yet: https://github.com/ipfs/go-libipfs/pull/32